pkg/action/bootentries.go

```diff
@@ -83,9 +83,9 @@ func selectBootEntrySystemd(cfg *config.Config, entry string) error {
 
 	}
 	originalEntries := entries
-	// when there are only 3 entries, we can assume they are either cos (which will be replaced eventually), fallback or recovery
-	if len(entries) == 3 {
```

Comment:
oooooh I just saw it, dumb of me, sorry about this :(

Reply:
no worries